packages/neon_lints/lint_maker.yaml

```diff
@@ -16,11 +16,12 @@ base:
         avoid_as: false
         avoid_catches_without_on_clauses: false
         diagnostic_describe_all_properties: false
+        document_ignores: false
```

Comment:
I feel like we should turn this on eventually.
Almost every time where the docs are missing it is a mistake and not done for a good reason.